app/assets/javascripts/media_player_wrapper/avalon_player_new.es6

```diff
@@ -671,7 +680,7 @@ class MEJSPlayer {
         let fullConfiguration = Object.assign({}, defaults, markerConfig);
         // Create a MediaElement instance
         this.player = new MediaElementPlayer(
-          `mejs-avalon-${this.mediaType}`,
+          'mejs-avalon-player',
```

Comment:
This is interesting.  Was the distinction between audio and video never being checked or used in the player code?

Reply:
We use the distinction when building the player, but it was not reflected in the player id